common/constants.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+const (
+	InternalDataSourcePath = "pkmn.csv"
```

Comment:
Keeping this kind of info separated is fine, although it's a lot more commonly found to have it on a config file, and then when the app initializes you would like to read the file with this and potentially more values, like the external api endpoints, db tables, etc...